rootfs/opt/qserv/k8s/kubectl/create-storage.sh

```diff
@@ -65,14 +67,10 @@ done
 
 DATA_ID=0
 
-if [ "$MINIKUBE" != true ]; then
-    kubectl apply -f "${DIR}/yaml/qserv-storageclass.yaml"
-fi
+kubectl apply -f "${DIR}/yaml/qserv-storageclass.yaml"
 for host in $MASTER $WORKERS;
 do
-    if [ "$MINIKUBE" != true ]; then
-        kubectl apply -f "${STORAGE_OUTPUT_DIR}/qserv-data-pv-${DATA_ID}.yaml"
-    fi
+    kubectl apply -f "${STORAGE_OUTPUT_DIR}/qserv-data-pv-${DATA_ID}.yaml"
     kubectl apply -f "${STORAGE_OUTPUT_DIR}/qserv-data-pvc-${DATA_ID}.yaml"
```

Comment:
Hmm, is this a typo? Why do we need to push two yaml configs named `-pv-` and `-pvc-`?